process_report/util.py

```diff
@@ -90,3 +92,8 @@ def fetch_s3(s3_filepath):
     invoice_bucket = get_invoice_bucket()
     invoice_bucket.download_file(s3_filepath, local_name)
     return local_name
+
+
+def fetch_nerc_rates(metadata_name, invoice_month):
```

Comment:
I don't see a good reason to have this function and unnecessarily bloat util.py. Just move `rates_info = load_from_url()` to the top of `processor_report.py`, and then just query the rate as needed instead of this function call.

Reply:
I have created a global `rates_info` in `process_report.py`